src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/spell/component/Knockback.java

```diff
@@ -24,7 +24,6 @@ public SpellCastResult invoke(ISpell spell, LivingEntity caster, Level level, Li
         Entity entity = target.getEntity();
         float velocity = ArsMagicaAPI.get().getSpellHelper().getModifiedStat(1, SpellPartStats.SPEED, modifiers, spell, caster, target);
         Vec3 delta = entity.getDeltaMovement();
-        // TODO ((LivingEntity) entity).knockback(); @IHH
```

Comment:
this implementation bypasses any knockback protection 
What about the method LivingEntity#knockback?

Reply:
This is the old implementation that allows y-knockback as well.